src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -109,4 +109,17 @@ public void blockingWrapperWorks() throws IOException {
             Matchers.equalTo(content)
         );
     }
+
+    @Test
+    public void move() throws Exception {
+        final byte[] data = "data".getBytes();
+        final BlockingStorage storage = new BlockingStorage(
+            new FileStorage(this.folder.newFolder().toPath())
+        );
+        final Key source = new Key.From("from");
+        storage.save(source, data);
+        final Key destination = new Key.From("to");
+        storage.move(source, destination);
+        MatcherAssert.assertThat(storage.value(destination), Matchers.equalTo(data));
+    }
```

Comment:
@olegmoz We should have tests for the other implementations of move() as well. Can you leave a TODO?

Reply:
@amihaiemil other implementations are trivial - they are just wrappers over this implementation. Anyway it makes it easy to break them... I created issue for that https://github.com/artipie/asto/issues/55